Gemfile

```diff
@@ -23,3 +23,13 @@ group :test do
   gem 'simplecov', '>= 0.11.2'
   gem 'yarjuf', '>= 2.0'
 end
+
+# Evaluate Gemfile.local if it exists
+if File.exists? "#{__FILE__}.local"
+  eval(File.read("#{__FILE__}.local"), binding)
+end
+
+# Evaluate ~/.gemfile if it exists
+if File.exists?(File.join(Dir.home, '.gemfile'))
+  eval(File.read(File.join(Dir.home, '.gemfile')), binding)
```

Comment:
Like I said in #180, I'm not generally a fan of the `Gemfile.local` pattern, but it's not something I feel strongly enough about to push back on.

BUT! You can eschew the passing of `binding` if you use `instance_eval` instead:

```
instance_eval(File.read("#{__FILE__}.local")
```